app/server/appsmith-plugins/openAiPlugin/src/main/resources/editor/vision.json

```
@@ -0,0 +1,113 @@
+{
+  "identifier": "VISION",
+  "controlType": "SECTION",
+  "conditionals": {
+    "show": "{{actionConfiguration.formData.command.data === 'VISION'}}"
+  },
+  "children": [
+    {
+      "label": "Models",
+      "tooltipText": "Select the model for response generation",
+      "subtitle": "ID of the model to use.",
+      "isRequired": true,
+      "propertyName": "vision_model_id",
+      "configProperty": "actionConfiguration.formData.visionModel.data",
+      "controlType": "DROP_DOWN",
+      "initialValue": "",
+      "options": ["gpt-4-vision-preview"],
+      "placeholderText": "All models will be fetched.",
+      "fetchOptionsConditionally": true,
+      "alternateViewTypes": ["json"],
+      "conditionals": {
+        "enable": "{{true}}",
+        "fetchDynamicValues": {
+          "condition": "{{true}}",
+          "config": {
+            "params": {
+              "requestType": "VISION_MODELS",
+              "displayType": "DROP_DOWN"
+            }
+          }
+        }
+      }
+    },
+    {
+      "label": "path to query",
+      "configProperty": "actionConfiguration.path",
+      "controlType": "INPUT_TEXT",
+      "initialValue": "/v1/chat/completions",
+      "isRequired": true,
+      "hidden": true
+    },
+    {
+      "label": "Max Token",
+      "tooltipText": "Max tokens to use for this assistant requests",
+      "Description": "Put a positive integer value",
+      "configProperty": "actionConfiguration.formData.maxTokens",
+      "controlType": "INPUT_TEXT",
+      "initialValue": "1",
```

```suggestion
      "initialValue": "16",
```
The initial value should be the same as the value open ai sets.